superset/assets/javascripts/explorev2/components/ControlPanelsContainer.jsx

```diff
@@ -119,6 +141,7 @@ function mapStateToProps(state) {
     alert: state.controlPanelAlert,
     isDatasourceMetaLoading: state.isDatasourceMetaLoading,
     fields: state.fields,
+    exploreState: state,
```

Comment:
do we need to pass the entire state object in here? could we pass just the keys we need?